jobs/kpi-forecasting/kpi_forecasting/configs/model_inputs/__init__.py

```diff
@@ -38,22 +38,3 @@ class ProphetHoliday:
     ds: List
     lower_window: int
     upper_window: int
-
-
-@attr.s(auto_attribs=True, frozen=False)
-class ModelConfig:
-
-    metric: str
-    slug: str
-    segment: Dict[str, str]
-    start_date: Optional[str] = None
-    holidays: Optional[pd.DataFrame] = None
-    regressors: Optional[List[ProphetRegressor]] = []
-    parameters: Optional[Dict[str, Union[List[float], float]]] = None
-    cv_settings: Optional[Dict[str, str]] = {
-        "initial": "365 days",
-        "period": "30 days",
-        "horizon": "30 days",
-        "parallel": "processes",
-    }
-    trend_change: Optional[str] = ""
```

Comment:
I assume this is just unused code? We're sure removing it won't break any other models we have here (e.g., DAU Desktop and DAU Mobile)?

Reply:
Correct, this is a class that I added that's no longer being used, so there's no risk to removing it.